wolfssl/sniffer.h

```diff
@@ -98,7 +98,8 @@ enum {
  * information associated with the SSL session.
  */
 
-#if defined(__IAR_SYSTEMS_ICC__) || defined(__GNUC__)
+#if (defined(__IAR_SYSTEMS_ICC__) && (__IAR_SYSTEMS_ICC__ > 8)) || \
```

Comment:
Is it not possible to include wolfssl/wolfcrypt/types.h here instead of redefining it?

Reply:
No point. Just delete that ifdef and the WOLFSSL_PACKED. It doesn't add anything to the structure that is tagged with it. I don't know why I added it.